ksp_plugin_test/test_plugin.hpp

```diff
@@ -0,0 +1,34 @@
+
+#pragma once
+
+#include <string>
+
+#include "astronomy/frames.hpp"
+#include "ksp_plugin/plugin.hpp"
+#include "physics/solar_system.hpp"
+
+namespace principia {
+namespace ksp_plugin {
+namespace internal_test_plugin {
+
+using astronomy::ICRFJ2000Equator;
+using physics::KeplerianElements;
+using physics::SolarSystem;
+
+class TestPlugin : public Plugin {
```

Comment:
Maybe a short comment for the constructor and the method.

Reply:
Done.